### ISSUE TITLE
Feature Request: Allow secondary easier methods to log in, such as a PIN

### ISSUE THREAD

**Issue by ddotthomas** [pop-os/cosmic-greeter#75]
I was thinking about how in the past I've seen people ask if there was any ways to enable a PIN log-in for Linux and I think it would help make the desktop even easier for people to use. 

I would probably limit it to just being used to log in and unlock any sessions already logged in. It would probably still be better if anything that requires root or 'sudo' still needs an admin password to fully authenticate.

**Comment by ChocolateLoverRaj** [comment]
I don't know any linux distros which have PIN login, which can be inconvenient for people who are used to Windows. It would be great of POP!_OS and COSMIC made PIN login possible.

I'm pretty sure that cosmic-greeter would have to support logging in with a PIN, but also there would need to be some way of setting and changing a PIN for a user.

**Comment by MadSpindel** [comment]
Yes, or maybe they can support Yubikey/Security key with PIN and then it would be very secure.

GNOME has some cool mockups:
https://gitlab.gnome.org/Teams/Design/os-mockups/-/commit/3c80eb9067a465100c7fcdada6f011e057cb5e58

**Comment by ChocolateLoverRaj** [comment]
Very cool. After seeing the unlock by QR code it made me think of Chrome OS and keeping your Chromebook unlocked when your phone is unlocked and nearby. It would be nice if COSMIC also had a phone-based unlock.

**Comment by ddotthomas** [comment]
To come back and add-on to this, a lot of end-users we support request that there's no password at all for logging in, for shared computers and machines that don't require as much security. I'm not sure how feasible this is and if PAM and the other linux authentication systems allow the account to not have a password.

**Comment by Layman806** [comment]
@ddotthomas Actually, gdm3 supports this kind of login method, and even cosmic-greeter does. Just that it works only for the first login. Subsequent attempts of logging in after locking the screen did not work.
I followed the instructions from here:
https://askubuntu.com/a/1304887
